eng/Generate.ps1

```diff
@@ -64,11 +64,17 @@ function Add-TestServer-Swagger ([string]$testName, [string]$projectSuffix, [str
 
 function Add-CadlRanch-Typespec([string]$testName, [string]$projectPrefix, [string]$cadlRanchProjectsDirectory) {
     $projectDirectory = Join-Path $cadlRanchProjectsDirectory $testName
+    $configPath = Join-Path $projectDirectory "tspconfig.yaml"
     if (Test-Path "$projectDirectory/*.sln") {
         $projectDirectory = Join-Path $projectDirectory "src"
     }
     $cadlMain = Join-Path $cadlRanchFilePath $testName "main.tsp"
-    Add-Typespec "$projectPrefix$testName" $projectDirectory $cadlMain
+    if (Test-Path $configPath) {
+        Add-Typespec "$projectPrefix$testName" $projectDirectory $cadlMain "--config=$configPath"
```

Comment:
Is `--config` a parameter of typespec compiler after 0.45.*? If so, this parameter is not recognized by 0.44, will it break our current pipeline for 0.44? Please make sure it does not break current pipeline.

Reply:
No, I have tested.